cellprofiler/gui/preferencesdlg.py

```diff
@@ -282,9 +281,16 @@ def get_preferences(self):
                 ["Intensity normalization",
                  cellprofiler.preferences.get_intensity_mode,
                  cellprofiler.preferences.set_intensity_mode,
-                 [cellprofiler.preferences.INTENSITY_MODE_RAW, cellprofiler.preferences.INTENSITY_MODE_NORMAL,
-                  cellprofiler.preferences.INTENSITY_MODE_LOG],
+                 [cellprofiler.preferences.INTENSITY_MODE_RAW,
+                  cellprofiler.preferences.INTENSITY_MODE_NORMAL,
+                  cellprofiler.preferences.INTENSITY_MODE_LOG,
+                  cellprofiler.preferences.INTENSITY_MODE_GAMMA],
                  cellprofiler.preferences.INTENSITY_MODE_HELP],
+                ["Intensity normalization factor",
+                 cellprofiler.preferences.get_normalization_factor,
+                 cellprofiler.preferences.set_normalization_factor,
+                 "Text",
```

Comment:
Should this be a different constant like `DIRBROWSE` or `CHOICE`, or is it fine to stay "Text"?

Reply:
I'll create a constant, for consistency.